### PR TITLE
docs(ecosystem): add missing plugins to core list

### LINKED ISSUE #5232: Ecosystem.md, missing @fastify/kafka, maybe other core plugins missing too
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the issue has not already been raised

### Issue

Add following core plugins to the ecosystem.md file

- [ ] @fastify/express
- [ ] @fastify/kafka
- [ ] @fastify/mysql
- [ ] @fastify/passport
- [ ] @fastify/swagger-ui
- [ ] @fastify/zipkin

### PR DESCRIPTION
**Pull request by Fdawgs**
closes #5232 

We were also missing the `@fastify/routes-stats` plugin.

#### Checklist
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -69,6 +71,8 @@ section.
   your HTTP requests to another server, with hooks.
 - [`@fastify/jwt`](https://github.com/fastify/fastify-jwt) JWT utils for
   Fastify, internally uses [fast-jwt](https://github.com/nearform/fast-jwt).
+- [`fastify/kafka`](https://github.com/fastify/fastify-kafka) Plugin to interact
+- with 
 - [`@fastify/leveldb`](https://github.com/fastify/fastify-leveldb) Plugin to
   share a common LevelDB connection across Fastify.
```
Comment (Fdawgs) [inline docs/Guides/Ecosystem.md:77]:
```suggestion
- [`@fastify/leveldb`](https://github.com/fastify/fastify-leveldb) Plugin to
  share a common LevelDB connection across Fastify.
- [`fastify/kafka`](https://github.com/fastify/fastify-kafka) Plugin to interact
  with Apache Kafka. 
```

File: docs/Guides/Ecosystem.md [hunk 2]
```diff
@@ -71,20 +73,26 @@ section.
   Fastify, internally uses [fast-jwt](https://github.com/nearform/fast-jwt).
 - [`@fastify/leveldb`](https://github.com/fastify/fastify-leveldb) Plugin to
   share a common LevelDB connection across Fastify.
+- [`fastify/kafka`](https://github.com/fastify/fastify-kafka) Plugin to interact
```
Comment (Fdawgs) [inline docs/Guides/Ecosystem.md:76]:
```suggestion
- [`@fastify/kafka`](https://github.com/fastify/fastify-kafka) Plugin to interact
```

File: docs/Guides/Ecosystem.md [hunk 3]
```diff
@@ -71,20 +73,26 @@ section.
   Fastify, internally uses [fast-jwt](https://github.com/nearform/fast-jwt).
 - [`@fastify/leveldb`](https://github.com/fastify/fastify-leveldb) Plugin to
   share a common LevelDB connection across Fastify.
+- [`@fastify/kafka`](https://github.com/fastify/fastify-kafka) Plugin to interact
+  with Apache Kafka.
```
Comment (Fdawgs) [inline docs/Guides/Ecosystem.md:77]:
```suggestion
- [`@fastify/kafka`](https://github.com/fastify/fastify-kafka) Plugin to interact
  with Apache Kafka.
- [`@fastify/leveldb`](https://github.com/fastify/fastify-leveldb) Plugin to
  share a common LevelDB connection across Fastify.
```

FML this is what I get for trying to do this on my phone.

File: docs/Guides/Ecosystem.md [hunk 4]
```diff
@@ -104,6 +112,8 @@ section.
   A simple plugin that enables response validation for Fastify.
 - [`@fastify/routes`](https://github.com/fastify/fastify-routes) Plugin that
   provides a `Map` of routes.
+- [`@fastify/routes-stats`](https://github.com/fastify/fastify-routes-stats) Provide stats
+  for routes using `node:perf_hooks`.
```
Comment (Fdawgs) [inline docs/Guides/Ecosystem.md:116]:
```suggestion
- [`@fastify/routes-stats`](https://github.com/fastify/fastify-routes-stats)
  Provide stats for routes using `node:perf_hooks`.
```

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**